src/metabase/driver/mongo/query_processor.clj

```diff
@@ -158,7 +161,26 @@
 
   DateTimeField
   (->rvalue [this]
-    (str \$ (->lvalue this)))
+    ;; omf: date/time fields using native Mongo operators
+    (let [unit (:unit this)
+      field (:field this)]
+      (case (or unit :default)
+        :default         (->rvalue field)
+        :minute          {:$dateToString {:format "%Y-%m-%d %H:%M" :date (->rvalue field)}}
```

Comment:
Normally you *would* use commas to separate key-value pairs on the same line.

So 

```clojure
{:format "%Y-%m-%d %H:%M", :date (->rvalue field)}
```

here.